Rakefile

```diff
@@ -1,9 +1,22 @@
 require 'cucumber/rake/task'
 require 'rspec/core/rake_task'
 require 'rubocop/rake_task'
+require 'data_mapper'
 
 RuboCop::RakeTask.new :cop
 RSpec::Core::RakeTask.new :spec
 Cucumber::Rake::Task.new :cuke
 
-task default: [:cop, :spec, :cuke]
\ No newline at end of file
+task default: [:cop, :spec, :cuke]
+
+task :auto_upgrade do
+  require './app/data_mapper_setup'
```

Comment:
I'd really like to find a better way to do this - to avoid using `require` within a task